src/pages/dataTableUtilities/reducer/cellReducers.js

```diff
@@ -26,6 +26,11 @@ export const refreshIndicatorRow = state => {
   return { ...state, indicatorRows: currentIndicator.rows };
 };
 
+export const focusCell = (state, action) => {
+  const { dataState } = state;
+  dataState.row[action.rowKey].focusedColumn = action.columnKey;
+};
```

Comment:
you would need to make `dataState` have a new reference by making it a new set, otherwise the equality check for rendering will return equal for the table. Then the `rowKey` for the specific row in the set will need a new reference, otherwise the row will not re-render

i.e.

```
const newDataState = new Map(dataState);
const rowState = newDataState.get(rowKey);
newDataState.set(rowKey, { ...rowState, isFocused: true });
```

I think this might be better in the `rowReducers.js` file as it's more for a row rather than a cell, what do you think?